src/constants.ts

```diff
@@ -1,9 +1,29 @@
+
 export interface ICallbacks {
   callbackName: string
   argsArr: Array<string>
   argsLength: number
 }
 
+
+export interface Version {
+  reactNativeSdkVersion: string
+  minSupportedIosVersion: string
+  maxSupportedIosVersion: string
+  androidSdkVersion: string
+  minSupportedAndroidVersion: string
+  maxSupportedAndroidVersion: string
+}
+
+export const reactNativeVersionObj: Version = {
+  reactNativeSdkVersion: '',
```

Comment:
`iosSdkVersion` is missing. Also, can we get these values directly from the SDK instead of hardcoding?

Reply:
i have update version method values , i think this is the way , or it will separate with bridge method . i can better explain on call .

Reply:
I read the same explanation on https://github.com/getphyllo/phyllo-connect-flutter/pull/25. This looks fine. Please make sure that we add this step to our release process that this file needs to be updated with each release.

Reply:
@erpankajpatel see this step is missed to update the Android SDK version number. That's why it's important to add in the release / update process. Do we have any such process document?

Reply:
@akshaybhalotia no we don't have any process document . i need to check once . if @chiragphyllo might be have .

Reply:
Yeah, we have this doc - https://getphyllo.atlassian.net/wiki/spaces/EN/pages/345473040/Front+end+Developers+release+check+list